Packs/Active_Directory_Query/Classifiers/classifier-User_Profile_-_Active_Directory_(Incoming).json

```diff
@@ -6,10 +6,6 @@
   "User Profile": {
    "dontMapEventToLabels": true,
    "internalMapping": {
-    " Street Address": {
```

Comment:
the space before the "Street" here should be removed but streetaddress incident field exist (under CommonTypes)

Reply:
there is already a Street address field in this classifier